.eslintrc.js

```diff
@@ -114,7 +114,10 @@ module.exports = {
       },
     ],
     '@typescript-eslint/prefer-optional-chain': 2,
-    '@typescript-eslint/prefer-nullish-coalescing': 2,
+    '@typescript-eslint/prefer-nullish-coalescing': [
+      2,
+      { ignoreMixedLogicalExpressions: true },
```

Comment:
I prefer not to add this generally

Reply:
Fair enough
But somethings gotta give

Add a nolint comment in the code then?
It won't pass eslint otherwise